### PR TITLE
[releng] Update to Spring Boot 2.5.2

### PR DESCRIPTION
**Pull request by pcdavid**
Signed-off-by: Pierre-Charles David <pierre-charles.david@obeo.fr>

### Type of this PR 

- [ ] Bug fix
- [ ] New feature or improvement
- [ ] Documentation
- [ ] Cleanup
- [ ] Test
- [ ] Build/Releng

### Issue(s)

...

### What does this PR do?

...

### Screenshot/screencast of this PR

...
 

### Potential side effects

...

### How to test this PR?

- [ ] Frontend Unit tests
- [ ] Backend Unit tests
- [ ] Cypress : please specify
- [ ] Manual Test : please specify

### Checklist

- [ ] I have read CONTRIBUTING carefully.
- [ ] I have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [ ] All my commits are signed-off (`-s`) with my mail address of my Eclipse Account.
- [ ] I have covered my changes by unit tests or integration tests or manual tests.
- [ ] All tests pass.
- [ ] I have updated the documentation accordingly
- [ ] New React components are availables in storybook

### CODE REVIEW

File: backend/sirius-web-test-diagrams-only-application/.classpath [hunk 1]
```diff
@@ -22,5 +21,29 @@
...
+	</classpathentry>
+	<classpathentry kind="src" output="target/test-classes" path="target/generated-test-sources/test-annotations">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="ignore_optional_problems" value="true"/>
+			<attribute name="m2e-apt" value="true"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
```
Comment (gcoutable) [inline backend/sirius-web-test-diagrams-only-application/.classpath:47]: Are these changes normal?

File: backend/sirius-web-test-sample-application/.classpath [hunk 2]
```diff
@@ -22,5 +21,29 @@
...
+	</classpathentry>
+	<classpathentry kind="src" output="target/test-classes" path="target/generated-test-sources/test-annotations">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="ignore_optional_problems" value="true"/>
+			<attribute name="m2e-apt" value="true"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
```
Comment (gcoutable) [inline backend/sirius-web-test-sample-application/.classpath:47]: Same thing here

**Review by gcoutable — COMMENTED** [review]
Some classpath changes may not be required.

File: backend/sirius-web-test-sample-application/.classpath (hunk 2 above)
Comment (pcdavid) [reply backend/sirius-web-test-sample-application/.classpath]: Done.

File: backend/sirius-web-test-diagrams-only-application/.classpath (hunk 1 above)
Comment (pcdavid) [reply backend/sirius-web-test-diagrams-only-application/.classpath]: No. Fixed.

**Review by gcoutable — APPROVED** [review]

**Pull request merged**